tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaIntegrationTest.java

```diff
@@ -156,8 +156,8 @@ private static void checkForErrorsInLogs(final String logs) {
     @BeforeClass
     @Override
     protected void setup() throws Exception {
-
         super.resetConfig();
+        this.conf.setEnableTransactionCoordinator(true);
```

Comment:
Is it necessary? The integration tests involve no transaction. If it requires transaction coordinator enabled, the design may have some problems.

Reply:
This is necessary, I found some clients' default read isolation is `read_committed`, so it needs to enable the transaction coordinator.

Reply:
Got it.